spec/translation_quality/account_settings.yaml

```diff
@@ -199,25 +408,150 @@ components:
                 data:
                   allOf:
                     - $ref: './tqc_common.yaml#/components/schemas/groupUidResponse'
+                    - $ref: './tqc_common.yaml#/components/schemas/severityLevelCodeResponse'
+                    - $ref: './tqc_common.yaml#/components/schemas/checkTypeCodeResponse'
+                    - type: object
+                      properties:
+                        subTypes:
+                          type: array
+                          items:
+                            allOf:
+                               - $ref: './tqc_common.yaml#/components/schemas/checkSubTypeCodeResponse'
+                               - $ref: './tqc_common.yaml#/components/schemas/severityLevelCodeResponse1'
```

Comment:
why do we need the both severityLevelCodeResponse1 and severityLevelCodeResponse?